crm_lead_product/report/crm_product_report.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (C) 2017-19 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from psycopg2.extensions import AsIs
+
+from odoo import fields, models, tools
+
+
+class ActivityReport(models.Model):
+    """CRM Lead Analysis"""
+
+    _name = "crm.product.report"
+    _auto = False
+    _description = "CRM Pipeline by Product Analysis"
+    _rec_name = "id"
+
+    active = fields.Boolean(readonly=True)
+    campaign_id = fields.Many2one("utm.campaign", "Campaing", readonly=True)
+    country_id = fields.Many2one("res.country", "Country", readonly=True)
+    company_id = fields.Many2one("res.company", "Company", readonly=True)
+    create_date = fields.Datetime(readonly=True)
+    date_closed = fields.Datetime("Closed Date", readonly=True)
+    date_conversion = fields.Datetime("Conversion Date", readonly=True)
+    date_deadline = fields.Datetime("Deadline Date", readonly=True)
+    date_open = fields.Datetime("Open Date", readonly=True)
+    lost_reason = fields.Many2one("crm.lost.reason", readonly=True)
+    name = fields.Char("Lead Name", readonly=True)
+    partner_id = fields.Many2one("res.partner", "Partner/Customer", readonly=True)
+    partner_name = fields.Char("Contact Name", readonly=True)
+    probability = fields.Float(group_operator="avg", readonly=True)
+    stage_id = fields.Many2one("crm.stage", "Stage", readonly=True)
+    team_id = fields.Many2one("crm.team", "Sales Team", readonly=True)
+    type = fields.Char(
+        selection=[("lead", "Lead"), ("opportunity", "Opportunity")],
+        help="Type is used to separate Leads and Opportunities",
+    )
```

Comment:
odoo odoo.fields: Field crm.product.report.type: unknown parameter 'selection', if this is an actual parameter you may want to override the method _valid_field_parameter on the relevant model in order to allow it 

i think there is a mistake with type of the field here?
```suggestion
    type = fields.Selection(
       [("lead", "Lead"), ("opportunity", "Opportunity")],
        help="Type is used to separate Leads and Opportunities",
    )
```

Reply:
Hello, thanks for your review, it is possible, i will have a look at it test it and provide feedback. Thank you one more time.

Reply:
> `odoo odoo.fields: Field crm.product.report.type: unknown parameter 'selection', if this is an actual parameter you may want to override the method _valid_field_parameter on the relevant model in order to allow it `
> 
> i think there is a mistake with type of the field here?

Funny thing is the same exists in 13.0